client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -97,9 +115,9 @@ const ConnectedSingleSiteJetpack = connectOptions(
 							<ConnectedThemesSelection
 								origin="wpcom"
 								defaultOption={ 'activate' }
-								secondaryOption={ 'tryandcustomize' }
+								secondaryOption={ config.isEnabled( 'jetpack/pijp' ) && hasUnlimitedPremiumThemes ? 'tryandcustomize' : '' }
```

Comment:
Can we drop this? `hideForTheme` should be smart enough to filter this for us I think.